extensions/resource-deployment/src/ui/modelViewUtils.ts

```diff
@@ -304,6 +311,7 @@ export function initializeWizardPage(context: WizardPageContext): void {
 				sectionInfo: sectionInfo
 			});
 		}));
+		hookUpDynamicEnablement(context);
```

Comment:
If you do this hookup on each page's onEnter method then you can make a field depend on fields on previous pages as well.

Reply:
That feature can be added if anyone actually needs it